tests/zzz_copy_sut_archive_files.rb

```diff
@@ -1,16 +1,15 @@
 require 'fileutils'
 
 step "Copy archive files from SUT" do
-  archive_files = ENV['SUT_ARCHIVE_FILES'].split(",")
-  job_name = ENV['PUPPET_GATLING_SIMULATION_ID']
-  Beaker::Log.notify("Copying #{archive_files.count} archive files from SUT")
-
-  archive_dir = "../puppet-gatling/#{job_name}/sut_archive_files"
-
-  FileUtils.mkdir_p(archive_dir)
-
-  archive_files.each do |s|
-    Beaker::Log.notify("Copying archive file '#{s}'")
-    scp_from(master, s, archive_dir)
-  end
+  today = Date.today().to_s
+  # truncate the job name so it only has the name-y part and no parameters
+  job_name = (ENV['JOB_NAME'] || 'unknown_jenkins_job')
```

Comment:
This seems to assume that you are in jenkins.  What about manual runs like when we are debugging a performance issue by tweaking the setup and rerunning the simulation?

Reply:
It just uses that to name the file. If there is no job name set, it defaults to 'unknown_jenkins_job' which is what you get when you run it outside of jenkins.

Reply:
right but the name implies it was still a jenkins job...   And all the reg exps don't need to be done on that known string either.